llama-index-core/llama_index/core/indices/prompt_helper.py

```diff
@@ -17,15 +17,18 @@
 from llama_index.core.bridge.pydantic import Field, PrivateAttr
 from llama_index.core.constants import DEFAULT_CONTEXT_WINDOW, DEFAULT_NUM_OUTPUTS
 from llama_index.core.llms.llm import LLM
+from llama_index.core.llms.structured_llm import StructuredLLM
 from llama_index.core.node_parser.text.token import TokenTextSplitter
 from llama_index.core.node_parser.text.utils import truncate_text
 from llama_index.core.prompts import (
     BasePromptTemplate,
     ChatPromptTemplate,
     SelectorPromptTemplate,
 )
+from llama_index.core.program.function_program import get_function_tool
```

Comment:
this leads to circular import issues for me

Reply:
Whoops

Reply:
This should be a more general util, it's super helpful -- will move this